src/libunicode/mktables.py

```diff
@@ -212,46 +212,38 @@ def __init__(self, _header_filename: str):
         self.file.write('\n')
         self.file.write('#include <libunicode/ucd_enums.h>\n')
         self.file.write('\n')
-        self.file.write('#include <fmt/format.h>\n')
+        self.file.write('#include <format>\n')
         self.file.write('\n')
-        self.file.write('namespace fmt\n{\n\n')
 
     def begin(self, _enum_class, _first):
         self.enum_class = 'unicode::' + _enum_class
         self.file.write('template <>\n')
-        self.file.write('struct formatter<{}>\n{{\n'.format(self.enum_class))
-
-        self.file.write('    template <typename ParseContext>\n')
-        self.file.write('    constexpr auto parse(ParseContext& ctx)\n')
-        self.file.write('    {\n')
-        self.file.write('        return ctx.begin();\n')
-        self.file.write('    }\n')
-
-        self.file.write('    template <typename FormatContext>\n')
-        self.file.write('    auto format({} value, FormatContext& ctx)\n'.format(self.enum_class))
+        self.file.write('struct std::formatter<{}>: std::formatter<std::string_view>\n{{\n'.format(self.enum_class))
+        self.file.write('    template <typename FormatContext>\n'.format(self.enum_class))
+        self.file.write('    auto format({} value, FormatContext& ctx) const -> FormatContext::iterator\n'.format(self.enum_class))
         self.file.write('    {\n')
+        self.file.write('        std::string_view name;\n')
         self.file.write('        switch (value)\n')
         self.file.write('        {\n')
         self.file.write('            // clang-format off\n')
 
     def member(self, _member):
         self.file.write(
-                '            case {0}::{1}: return fmt::format_to(ctx.out(), "{2}");\n'.format(
+            '            case {0}::{1}: name = "{2}"; break;\n'.format(
             self.enum_class, sanitize_identifier(_member), _member)
         )
 
     def end(self):
         self.file.write('            // clang-format off\n')
         self.file.write("        }\n")
-        self.file.write('        return fmt::format_to(ctx.out(), "({})", unsigned(value));\n')
+        self.file.write('        return formatter<string_view>::format(name, ctx);\n')
```

Comment:
Can it deduce template argument for `formatter` here ?

Reply:
@Yaraslaut the template argument for formatter is string_view here. Why would it not be?

Reply:
Ok, so definitely was thinking about something else while writing this comment xD 

Idea behind comment is to avoid writing `formatter<string_view>` but i failed to notice that this is inside custom formatter